.shippable.yml

```diff
@@ -55,6 +62,8 @@ build:
             fi;
             echo "- Verify commit message and coding style";
             ./scripts/ci/check-compliance.py || true;
+          elif [ "$IS_PULL_REQUEST" = "true" ]; then
+            export ARCH="-a arm -a arc -a nios2 -a riscv32";
```

Comment:
I do not know how we can miss an arch, we are not adding architectures on a daily basis. The hardcoding is here based on timing and numbers we see when running the sanitycheck in multiple slaves in parallel. We have to do this for better distribution of the load and to get results faster. Sure we can go wild and make sanitycheck script do that in a programmatic way, but thats what we have now. Every minute counts when running sanitycheck and based on monitoring 100s of jobs, this will help.